index.html

```diff
@@ -1,315 +1,357 @@
-  <title>Portfolio</title>
-</head>
-<body>
-  <section class="headersec">
-    <div class="topnav-logo">
-      <h4>My Logo</h4>
-    </div>
-    <div class="menu-icon-wrapper">
-      <div class="menu-icon" onclick="toggleCssMenu(this)">
-        <div class="three-line">
-          <span></span>
-          <span></span>
-          <span></span>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header id="header">
+      <div class="header-container">
+        <div class="logo">
+          <a href="#">My Logo</a>
         </div>
-      </div>
-    </div>
-    <div id="cssmenu">
-
-      <ul>
-        <li><a href="#">Contact</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Potfolio</a></li>
-      </ul>
-    </div>
-    <div class="about-flex-container">
-      <div class="name">
-        <h1>I’m Lilian,<br>
-          Glad to see you!</h1>
-      </div>
-      <div class="description">
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
-          and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      </div>
-      <div class="connectt">
-        <h5>LET’S CONNECT</h5>
-      </div>
-      <div class="social-icons">
-        <ul class="social-icons">
-          <li class="social-icon">
-            <a href="https://twitter.com/ifnotlily"> <img src="./images/twitter.svg" alt="Lilian's Twitter" /> </a>
-          </li>
-          <li class="social-icon">
-            <a href="https://www.linkedin.com/in/lilian-francis/">
-              <img src="./images/Linkedin.svg" alt="Lilian's LinkedIn" />
-            </a>
-          </li>
-          <li class="social-icon">
-            <a href="https://medium.com/@chatwitlily/">
-              <img src="./images/medium.svg" alt="Lilian's Medium" /> </a>
-          </li>
-          <li class="social-icon">
-            <a href="https://angel.co/u/lilian-francis"> <img src="./images/angellist.svg" alt="Lilian's AngelList" />
-            </a>
-          </li>
-          <li class="social-icon">
-            <a href="https://github.com/ifnotlily/">
-              <img src="./images/github.svg" alt="Lilian's Github" />
-            </a>
-          </li>
-        </ul>
-      </div>
-    </div>
-  </section>
-  <section class="nature-section">
-    <div class="natureimg">
-
-    </div>
-    <div class="section-grid">
-      <div id="item-0">
-      </div>
-      <div id="item-1">
-        <h3>Tonic</h3>
```

Comment:
The image for the project should have a full container width on medium viewport. Moreover, tech stack info is also missing. Please see the below screenshot for reference.
![project1](https://user-images.githubusercontent.com/35504149/130117427-be3cf5d8-2897-41ea-aa2f-1f2d6ee2fbcd.png)